Week2/homework/maartjes-work.js

```diff
@@ -45,21 +45,32 @@ const tuesday = [
 const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
-function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
-}
-
-// eslint-disable-next-line no-unused-vars
-const earnings = computeEarnings(maartjesTasks, maartjesHourlyRate);
-
+// Start: Adham's code
+const maartjesDurationsHours = []; /* Collect our data in one array */
+maartjesTasks.forEach(element => {
```

Comment:
Ah - I wonder why you opted for this approach rather than using `map` and `filter` functions here. There was a hint in the homework itself when it was stated - 

```
- Map the tasks to durations in hours.
- Filter out everything that took less than two hours (i.e., remove from the collection)
```

Reply:
This is my way in life and pc. First collect data, give it the desired case then analyse it. And as what you said it was a hint, and not a conditional step.
 A condition while I am collecting  the data so I pickup just what I need. Filter wil not help in this way.
Unfortunately , almost all the steps sound conditional steps. That is what I got form the Unit Test.